server/database/mock_database.py

```diff
@@ -23,14 +24,35 @@ def insert(topic, summary, source):
         with open(DATABASE_NEGATIVE_PATH, "w") as mock_db:
             yaml.dump(data, mock_db, default_flow_style=False)
 
+    @staticmethod
+    def insert_new_clusters(cluster_dict):
+        data = DatabaseNegative.__get_data()
+        data['clusters'] = cluster_dict
+        with open(DATABASE_NEGATIVE_PATH, "w") as mock_db:
+            yaml.dump(data, mock_db, default_flow_style=False)
+
+
     @staticmethod
     def get_summaries_by_topics():
         return DatabaseNegative.__get_data()['articles']
 
+    @staticmethod
+    def get_all_summaries():
+        article_dict = DatabaseNegative.__get_data()['articles']
+        summaries = list()
+        for topic in article_dict.keys():
+            summaries += article_dict[topic]
+        return summaries
```

Comment:
flatten `article_dict.values()` with list comprehension: `return [article for group in article_dict.values() for article in group]`